samples/msgext-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,102 @@
+---
+page_type: sample
+description: This sample demonstrates how to integrate Azure AI Search in a Teams message extension to compare a contract proposal against a guidance policy checklist to see if the proposal complies with the guidance. It uses Teams Toolkit for Visual Studio Code and JavaScript, and the message extension can be used as a plugin in Microsoft Copilot for Microsoft 365.
```

Comment:
This sample name should be "msgext-ai-doc-compliance-checker"
Pawan has added one new section in root readme, so this should also be linked to that section in root readme.
Refer his PR.

Reply:
Fixed